extensions/adobe/experience/implementations.schema.json

```diff
@@ -28,6 +28,11 @@
       "$ref": "https://ns.adobe.com/xdm/context/implementationdetails",
       "description":
         "Client details interacting with Adobe Experience Cloud ID Service."
+    },
+    "https://ns.adobe.com/experience/adcloud": {
+      "title": "Client Implementation for Adobe AdCloud",
+      "$ref": "https://ns.adobe.com/xdm/context/implementationdetails",
+      "description": "Client details interacting with Adobe AdCloud."
```

Comment:
No references to specific Adobe products

Reply:
Other adobe-specific references exist here. Should we move all of them out then?

Reply:
Didn't realize this was inside an extension schema - this is fine.

Reply:
I believe all concerns are now addressed from these comments.